<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vul_introduction_window</class>
 <widget class="QWidget" name="vul_introduction_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>漏洞-介绍编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_introduction">
     <property name="text">
      <string>漏洞介绍</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_introduction"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_button" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_yes">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_no">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
